src/main/features/dashboard/routes/claim-details.ts

```diff
@@ -0,0 +1,29 @@
+import * as express from 'express'
+import { Paths as DashboardPaths } from 'dashboard/paths'
+import ClaimStoreClient from 'claims/claimStoreClient'
+import Claim from 'app/claims/models/claim'
+import ErrorHandling from 'common/errorHandling'
+
+async function renderView (req: express.Request, res: express.Response): Promise<void> {
+
+  try {
+    const claim: Claim = await ClaimStoreClient.retrieveByClaimReference(res.locals.user.dashboardDaft.document.search.reference, res.locals.user.bearerToken)
+    console.log(claim)
```

Comment:
update tslint.json file to ban console.logs:
https://github.com/hmcts/cmc-citizen-frontend/blob/master/tslint.json#L5

Reply:
done, cheers :)